test/core/componentGroupTests.ts

```diff
@@ -65,7 +65,8 @@ describe("ComponentGroups", () => {
     componentGroup.remove(c1);
     assert.deepEqual(componentGroup.components(), [c0, c2],
     "removing a Component not in the Group does not remove Components from the Group");
-    assert.strictEqual(svg.node().childNodes[0], (<D3.Selection> (<any> c1)._element).node(), "The Component not in the Group stayed put");
+    assert.strictEqual((<Node> svg.node()).childNodes[0], (<d3.Selection<void>> (<any> c1)._element).node(),
```

Comment:
The test code doesn't know that though, because `_element` isn't public. After we `<any>`-cast c1, any fields on it are also `<any>`.

That said, it's probably better just to rewrite this `assert`.